include/crocoddyl/multibody/actions/contact-fwddyn.hxx

```diff
@@ -192,15 +192,17 @@ void DifferentialActionModelContactFwdDynamicsTpl<Scalar>::quasiStatic(
       pinocchio::rnea(pinocchio_, d->pinocchio, q, VectorXs::Zero(state_->get_nv()), VectorXs::Zero(state_->get_nv()));
 
   contacts_->calc(d->multibody.contacts, x);
+  // Allocates memory
   MatrixXs jc_view = d->multibody.contacts->Jc.template leftCols<6>();
```

Comment:
Why to not use a view directly?

Reply:
I couldn't get `Eigen::Map`, or `Eigen::Block` to work with `JacobiSVD` module.